helpers/filterSeats.js

```diff
@@ -104,7 +104,7 @@ module.exports = function(dust) {
       _.forEach(params.bandColours, function(sectionValue, sectionKey) {
         // check for match between transformer and existing data
         // or check for section that might not have full title ( e.g. Grand Circle in Grand Circle (Left) )
-        if (ticketRatesSection === sectionKey ) {
+        if (ticketRatesSection === sectionKey || ticketRatesSection.indexOf(sectionKey) > -1) {
```

Comment:
Actually, just looking at this again... why would this not always be an array?
`if (ticketRatesSection === sectionKey || ticketRatesSection.indexOf(sectionKey) > -1) {`

could be just
`if (ticketRatesSection.indexOf(sectionKey) > -1) {`

If you turn it into an array using:
`ticketRatesSection = Array.prototype.concat( ticketRatesSection )`

at the beginning of this function.